pkg/server/smtp/listener.go

```diff
@@ -72,12 +74,26 @@ func NewServer(
 	manager message.Manager,
 	apolicy *policy.Addressing,
 ) *Server {
+
+	tlsConfig := &tls.Config{}
+	if smtpConfig.TLSEnabled {
+		var err error
+		tlsConfig.Certificates = make([]tls.Certificate, 1)
+		tlsConfig.Certificates[0], err = tls.LoadX509KeyPair(smtpConfig.TLSCert, smtpConfig.TLSPrivKey)
+		if err != nil {
+			log.Error().Msgf("Failed loading X509 KeyPair: %v", err)
```

Comment:
Prefix `.Str("module", "smtp").Str("phase", "startup")` to these log statements, or create a scoped logger similar to Start() below:
```
slog := log.With().Str("module", "smtp").Str("phase", "startup").Logger()
```